NEWS.md

```diff
@@ -27,6 +27,12 @@
 
 ### What's changed
 
+-   For compatibility with CRAN, the number of threads used by Polars may be automatically set to 2 (#720).
+    To disable this behavior and have the maximum number of threads used automatically,
+    one of the following ways can be used:
+    -   Build the Rust library with the `disable_auto_limit_max_threads` feature.
+    -   Set the `polars.disable_auto_limit_max_threads` option to `TRUE` with the `options()` function
+        before loading the package.
```

Comment:
We could mention `Sys.setenv(POLARS_MAX_THREADS =)` so that users know that they don't have only the choice of 2 threads or all of them, no?

Reply:
How about 87d5eebaa8ba8e04cedab5827c6c3277bdc006fc?